### DISCUSSION TITLE
Company Scrip and Crypto Tokens

### DISCUSSION THREAD

**Discussion by rufuspollock** [rufuspollock/awesome-crypto-critique#81] [Ideas]
Company scrip seems a useful analogy for crypto tokens. Would be interesting to explore this analogy and write it up as it probably provides some useful lessons. Also noteworthy that company scrip was largely banned (as exploitative IIRC).

https://en.wikipedia.org/wiki/Company_scrip

> Company [scrip](https://en.wikipedia.org/wiki/Scrip) is scrip (a substitute for government-issued [legal tender](https://en.wikipedia.org/wiki/Legal_tender) or [currency](https://en.wikipedia.org/wiki/Currency)) issued by a [company](https://en.wikipedia.org/wiki/Company) to [pay](https://en.wikipedia.org/wiki/Payroll) its employees. It can only be exchanged in [company stores](https://en.wikipedia.org/wiki/Company_store) owned by the employers.[[1]](https://en.wikipedia.org/wiki/Company_scrip#cite_note-David_Ginsburg-1)[[2]](https://en.wikipedia.org/wiki/Company_scrip#cite_note-Taylor_1951-2)[[3]](https://en.wikipedia.org/wiki/Company_scrip#cite_note-Wis_ref-3) In the [United Kingdom](https://en.wikipedia.org/wiki/United_Kingdom), such [truck systems](https://en.wikipedia.org/wiki/Truck_system) have long been formally outlawed under the [Truck Acts](https://en.wikipedia.org/wiki/Truck_Act). In the United States, payment in scrip became illegal in 1938 as part of the [Fair Labor Standards Act](https://en.wikipedia.org/wiki/Fair_Labor_Standards_Act).[[4]](https://en.wikipedia.org/wiki/Company_scrip#cite_note-4)
> 
> In the [United States](https://en.wikipedia.org/wiki/United_States), [mining](https://en.wikipedia.org/wiki/Mining) and [logging camps](https://en.wikipedia.org/wiki/Logging_camp) were typically created, owned and operated by a single company.[[5]](https://en.wikipedia.org/wiki/Company_scrip#cite_note-Green-5) These locations, some quite remote, were often cash poor;[[1]](https://en.wikipedia.org/wiki/Company_scrip#cite_note-David_Ginsburg-1)[[2]](https://en.wikipedia.org/wiki/Company_scrip#cite_note-Taylor_1951-2)[[3]](https://en.wikipedia.org/wiki/Company_scrip#cite_note-Wis_ref-3) even in ones that were not, workers paid in scrip had little choice but to purchase goods at a company store, as exchange into currency, if even available, would exhaust some of the value via the exchange fee. With this economic [monopoly](https://en.wikipedia.org/wiki/Monopoly), the employer could place large [markups](https://en.wikipedia.org/wiki/Markup_(business)) on goods, making workers dependent on the company, thus enforcing employee "[loyalty](https://en.wikipedia.org/wiki/Loyalty)".[[5]](https://en.wikipedia.org/wiki/Company_scrip#cite_note-Green-5)[[6]](https://en.wikipedia.org/wiki/Company_scrip#cite_note-6) While scrip was not exclusive to the coal industry, an estimated 75 percent of all scrip used was by coal companies in Kentucky, Virginia, and West Virginia.[[7]](https://en.wikipedia.org/wiki/Company_scrip#cite_note-7) Because of this, many derived nicknames for the type of currency originated in the Appalachian mining communities, such as "Flickers," "Clackers," and "Dugaloos."[[8]](https://en.wikipedia.org/wiki/Company_scrip#cite_note-Coal_Mine_Script-8)
> 
> Tokens were made out of a variety of metals, including brass, copper, zinc, and nickel.[[8]](https://en.wikipedia.org/wiki/Company_scrip#cite_note-Coal_Mine_Script-8) There were additionally "compressed fibre" coins produced during World War II in an effort to conserve metals for wartime production.[[8]](https://en.wikipedia.org/wiki/Company_scrip#cite_note-Coal_Mine_Script-8)

**Comment by stevesong** [comment]
Might be good to contrast these with Local Exchange Trading Systems (LETS) which for the most part have been positive   https://en.wikipedia.org/wiki/Local_exchange_trading_system  

Sidebar:  Tim Jenkin, chief cryptographer for the ANC back in the day, profiled in the movie Escape from Pretoria, is also the founder of the South African Community Exchange System (https://www.community-exchange.org/home/who-are-we/).  He would be an interesting person to interview for his views on alternative currencies.

**Reply by rufuspollock** [reply]
@stevesong I believe LETS and community currencies have a pretty strong connection (?). What have been the successes of LETS and what are their weaknesses?

BTW we created a mini LETS for an event back in 2005 http://webarchive.okfn.org/okfn.org/201404/events/wsfii/

**Reply by joannebitcoin** [reply]
Let's see these examples,

https://www.citycoins.co/miamicoin
https://www.citycoins.co/nyccoin

The weaknesses is you just can mint them but not trade. It cannot be trade because CITIES doesn't have BIT Licenses

https://en.wikipedia.org/wiki/BitLicense

Solution —>  Let's allocate officially in PLANET EARTH DAO 

https://ipfs.fleek.co/ipfs/bafybeibqpfekg34xgwde7isoyc5eo6luyyiwlqkwc5qig2sgn7am46tpyu

We can allocate not only city coins but Nation coins, regions and worldwide...
